### PR TITLE
BugFix/NONE/fix-orbit-popup-button-positioning

### PR DESCRIPTION
**Pull request by ksanmartin909**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1309,11 +1309,6 @@ Also, some elements appear to overlap, or be awkwardly placed when viewing alone
     font-size: 16px;
   }
 
-  .earth_orbits_contaienr {
-    min-height: unset;
-    min-width: unset;
-  }
-
```
Comment (ksanmartin909) [inline src/App.css:1316]: `container` is spelled wrong anyway so this  clearly isn't doing anything

File: src/App.css [hunk 2]
```diff
@@ -1361,25 +1356,22 @@ Also, some elements appear to overlap, or be awkwardly placed when viewing alone
...
     width: var(--mobile-view-width);
     left: 0%;
-    height: 35%;
-    min-height: 340px;
+    height: 40%;
     top: 0;
-    display: flex;
-    flex-direction: column;
-    align-self: start;
-    justify-content: flex-start;
```
Comment (ksanmartin909) [inline src/App.css:1370]: I pretty much returned all of the popup properties back to what they were on master and made adjustments around it.

File: src/App.css [hunk 3]
```diff
@@ -1665,8 +1660,9 @@ Also, some elements appear to overlap, or be awkwardly placed when viewing alone
 }
 
 
-/* Phones (Portrait) */
-@media (max-width: 500px) {
+/* Phones (Portrait)*/
+/* Desktop */
+@media (max-width: 740px) {
```
Comment (ksanmartin909) [inline src/App.css:1665]: Forces the desktop to switch to a vertical layout a bit sooner.

File: src/App.css [hunk 4]
```diff
@@ -2056,13 +2052,4 @@ Also, some elements appear to overlap, or be awkwardly placed when viewing alone
...
   }
-}
-
-@media (max-height: 640px) {
-
-  /* Shrink popup container for shorter height devices */
-  .orbit_title_popup_container {
-    height: 45%;
-    top: 51%;
-  }
```
Comment (ksanmartin909) [inline src/App.css:2067]: This fixed something at some point but is now getting in the way.

**Review by rraaschfilament — APPROVED** [review]

**Pull request merged**